src/ansys/fluent/core/workflow.py

```diff
@@ -336,6 +412,27 @@ def _task_by_id(self, task_id):
         return self._task_by_id_impl(task_id, workflow_state)
 
 
+class ExtendedWorkflow(WorkflowWrapper):
+
+    def __init__(self, workflow, command_source):
+        super().__init__(workflow, command_source)
+
+    def __getattr__(self, attr):
+        try:
+            result = super().__getattr__(attr)
+            if result:
+                return result
+        except AttributeError:
+            pass
+        child_tasks = self.ordered_children()
```

Comment:
delegate to a method whose name describes _find the loaded task in the current workflow whoise command has a help string attribute matching attr_